config/sync/core.entity_view_display.node.stanford_opportunity.default.yml

```diff
@@ -70,14 +72,50 @@ third_party_settings:
                   field_label:
                     label_value: ''
                     label_tag: ''
-            weight: 0
+            weight: 2
             additional: {  }
-          e712f81b-4aa1-4bc7-bb7f-5068acb3bbd1:
-            uuid: e712f81b-4aa1-4bc7-bb7f-5068acb3bbd1
+          d2e4901d-e5fa-4d36-9bc0-2ff0545eb32b:
+            uuid: d2e4901d-e5fa-4d36-9bc0-2ff0545eb32b
             region: main
             configuration:
-              id: 'field_block:node:stanford_opportunity:body'
-              label: Body
+              id: 'inline_block:stanford_component_block'
```

Comment:
Inline blocks aren't supported as default configuration. You probably are looking for the "Main Content Anchor Target"
![image](https://github.com/user-attachments/assets/663bc874-b4aa-4b9a-b24f-67d1b35368ca)